<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchmarkWindow</class>
 <widget class="QMainWindow" name="BenchmarkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="VescConnectorWidget" name="mVescConnector1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="VescConnectorWidget" name="mVescConnector2" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBenchmarkPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abortPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 45, 55);
color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="benchmarkSettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>262</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Drive Current</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="driveCurrentBox">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max Brake Current</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="brakeCurrentMaxBox">
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Brake Current Increment</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="brakeCurrentIncrementBox">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="mTextLog">
    <property name="geometry">
     <rect>
      <x>583</x>
      <y>218</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveResultsPushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRpmLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VescConnectorWidget</class>
   <extends>QWidget</extends>
   <header>vescconnectorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
